src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -1813,8 +1823,11 @@ void rtklib_pvt_gs::initialize_and_apply_carrier_phase_offset()
 int rtklib_pvt_gs::work(int noutput_items, gr_vector_const_void_star& input_items,
     gr_vector_void_star& output_items __attribute__((unused)))
 {
+    std::cout << "\nNumber of records in iteration: " << noutput_items;
```

Comment:
did you mean to not newline terminate this output?

See above comment about `std::cout` in general and stream flushing.